experiment/adscert/SignerLogger.go

```diff
@@ -0,0 +1,38 @@
+package adscert
+
+import (
+	"fmt"
+	"github.com/golang/glog"
+)
+
+type SignerLogger struct {
+}
+
+func (sl *SignerLogger) Debugf(format string, args ...interface{}) {
+	//there is no Debug level in glog
+	glog.Infof(format, args...)
```

Comment:
Is there any advantage of using `glog` vs the default level logger that comes with the `adscert/logger` package?